hbase-resource-bundle/src/main/resources/supplemental-models.xml

```diff
@@ -2251,4 +2251,22 @@ Copyright (c) 2007-2017 The JRuby project
       </licenses>
     </project>
   </supplement>
+  <supplement>
+    <project> <!-- v3.4.0 comes in via HADOOP-17317 -->
+      <groupId>dnsjava</groupId>
+      <artifactId>dnsjava</artifactId>
+      <version>3.4.0</version>
+      <name>dnsjava</name>
+      <licenses>
+        <license>
+          <name>BSD 2-Clause License</name>
+          <url>https://opensource.org/licenses/BSD-2-Clause</url>
+          <distribution>repo</distribution>
+          <comments>
+            Copyright (c) 1998-2011, Brian Wellington.
+          </comments>
```

Comment:
the BSD 2 Clause license is not in the set of licenses where we aggregate all the dependencies provided with a single copy of the license text. (something that happens in hbase-resource-bundle/src/main/resources/META-INF/LICENSE.vm )

in such cases, we rely on the `comments` field here to provide the entirety of what we'll put into the generated LICENSE file. so this needs to include both this copyright statement and the complete text of the BSD 2 clause.

essentially everything from the LICENSE file on the dependency: https://github.com/dnsjava/dnsjava/blob/v3.4.0/LICENSE

it's the same kind of thing we had to do for jamon-runtime for the  MPL.